### PR TITLE
[Docs] Add release note for v1.3.0

### PR DESCRIPTION
**Pull request by abuccts**
Add release note for v1.3.0.

### CODE REVIEW

File: RELEASE_NOTE.md [hunk 1]
```diff
@@ -1,5 +1,24 @@
...
+- Marketplace
+  - New templates in marketplace (https://github.com/microsoft/openpaimarketplace/issues/60)
+- HiveD Scheduler
+  - Support cluster autoscale with HiveD scheduler on AKS (#4868)
+  - Support dynamic sku types for different vc on webportal (#4900)
+- Advanced job debug mode
+  - Add per task retry history (https://github.com/microsoft/frameworkcontroller/pull/62, #4958, #4966)
+  - Expose Kubernetes events (#4939, #4975)
+- GPU monitoring and utilization
+  - Support job tagging (#4853)
```
Comment (suiguoxin) [inline RELEASE_NOTE.md:14]: - Support job tagging #4924 
 - Stop low GPU utilization job with alert-manager #4940 
-  typo:  with -> when

File: RELEASE_NOTE.md [hunk 2]
```diff
@@ -1,5 +1,24 @@
...
+  - Cordon node with GPU ECC Errors (#4942)
```
Comment (suiguoxin) [inline RELEASE_NOTE.md:15]: - Support job tagging #4924 
 - Stop low GPU utilization job with alert-manager #4940 
-  typo:  with -> when

Comment (abuccts) [reply RELEASE_NOTE.md:16]: > typo: with -> when

It should be "*with*", or "Cordon node *when* it has GPU ECC Errors"

**Review by Binyang2014 — APPROVED** [review]

**Review by debuggy — APPROVED** [review]

File: RELEASE_NOTE.md [hunk 3]
```diff
@@ -1,5 +1,25 @@
...
 
+## Nov 2020 (version 1.3.0)
+
+- Marketplace
+  - New templates in marketplace (https://github.com/microsoft/openpaimarketplace/issues/60)
+- HiveD Scheduler
+  - Support cluster autoscale with HiveD scheduler on AKS (#4868)
+  - Support dynamic sku types for different vc on webportal (#4900)
+- Advanced job debug mode
+  - Add per task retry history (https://github.com/microsoft/frameworkcontroller/pull/62, #4958, #4966)
```
Comment (yqwang-ms) [inline RELEASE_NOTE.md:11]: These issue link are not showed as hyperlink.

Comment (abuccts) [reply RELEASE_NOTE.md:11]: not sure whether it's a regression on markdown rendering, as previous release notes all used this format

Comment (abuccts) [reply RELEASE_NOTE.md:11]: this is not a breaking issue for the release, I will send a docs fix in another PR

**Pull request merged**